packages/adapter-d1/src/d1.ts

```diff
@@ -176,6 +203,11 @@ export class PrismaD1 extends D1Queryable<StdClient> implements DriverAdapter {
     const tag = '[js::startTransaction]'
     // console.debug(`${tag} options: %O`, options)
 
+    this.warnOnce(
+      'D1 Transaction',
+      "Cloudflare D1 - currently in Beta - does not support transactions. When using Prisma's D1 adapter, implicit & explicit transactions will be ignored and ran as individual queries, which breaks the guarantees of the ACID properties of transactions. For more details see https://pris.ly/d/d1-transactions",
```

Comment:
Added a task to create `https://pris.ly/d/d1-transactions`

https://github.com/prisma/team-orm/issues/895